### ISSUE TITLE
Unreachable error after parsing about 1mb of json

### ISSUE THREAD

**Issue by andreievg** [extism/js-pdk#107]
extism-js version 1.2.0

Getting this error when parsing about 1mb of json data:

```
 0: 0xc3ec8 - <unknown>!<wasm function 1207>
    1: 0x4e85e - <unknown>!<wasm function 439>
    2: 0x4f2be - <unknown>!<wasm function 447>
    3: 0x82d3d - <unknown>!<wasm function 769>
    4: 0x3f736 - <unknown>!<wasm function 426>
    5: 0x40633 - <unknown>!<wasm function 434>
    6: 0x42652 - <unknown>!<wasm function 434>
    7: 0x29876 - <unknown>!<wasm function 263>
    8: 0xd24b8 - <unknown>!<wasm function 1347>
    9: 0xd3e76 - <unknown>!<wasm function 1364>

Caused by:
    wasm trap: wasm `unreachable` instruction executed
```

Here is repo for replication. Works fine with rust-pdk

https://github.com/andreievg/Extism-js-pdk-error-107

**Comment by bhelx** [comment]
Thank you for the reproducible project! I will take a look

**Comment by bhelx** [comment]
Which version of extism-js are you using @andreievg ?

I was able to repro, but then when trying the `main` branch the problem went away. I'll push another release and see if that clears it up in the meantime.

**Comment by bhelx** [comment]
I'll keep exploring, but i suspect the upgrade to rquickjs may have fixed a memory bug https://github.com/extism/js-pdk/pull/88

**Comment by bhelx** [comment]
Released 1.3.0 https://github.com/extism/js-pdk/releases/tag/v1.3.0

**Comment by andreievg** [comment]
Thanks for your help, it worked but cause another issue, i figured it, would be great to get your input: https://github.com/extism/js-pdk/issues/108
